### PR TITLE
clean up esc_url and unnecessary var

### PR DESCRIPTION
**Pull request by ggwicz**
_Ref:_ [C#39176](https://central.tri.be/issues/39176)

### CODE REVIEW

File: src/Tribe/Amalgamator.php [hunk 1]
```diff
@@ -258,8 +258,8 @@ private function delete_posts( $post_ids ) {
 	 */
 	public static function migration_button( $text = '' ) {
 		$text     = $text ? $text : __( 'Merge Duplicates', 'tribe-events-calendar' );
-		$html     = '<a href="%s" class="button">%s</a>';
 		$settings = Tribe__Events__Settings::instance();
+
```
Comment (barryhughes) [inline src/Tribe/Amalgamator.php]: Super minor: codesniffer is flagging up the unnecessary whitespace here (two tabs).

File: src/Tribe/Amalgamator.php [hunk 2]
```diff
@@ -269,11 +269,11 @@ public static function migration_button( $text = '' ) {
 				), admin_url( 'edit.php' )
 			)
 		);
-		$url  = esc_url( add_query_arg( array( 'amalgamate' => '1' ), $url ) );
+
```
Comment (barryhughes) [inline src/Tribe/Amalgamator.php]: Super minor: codesniffer is flagging up the unnecessary whitespace here (two tabs).

**Comment by barryhughes** [comment]
:8ball:

**Comment by zbtirrell** [comment]
Missing ticket reference
> via jenkins/no-ticket-ref
